tests/integration_tests/test_class_imbalance_feature.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from ludwig.api import LudwigModel
+from ludwig.backend import LocalBackend
+
+
+def run_test_imbalance(
+    input_df,
+    config,
+    balance,
+):
+    model = LudwigModel(config)
+    _, output_dataset, output_dir = model.train(
+        input_df,
+        skip_save_model=True,
+        skip_save_log=True,
+        skip_save_progress=True,
+        skip_save_processed_input=True,
+        skip_save_training_description=True,
+        skip_save_training_statistics=True,
+    )
+
+    input_train_set = input_df.sample(frac=0.7, replace=False)
+    processed_len = output_dataset[0].size
+    processed_target_pos = sum(output_dataset[0].dataset["Label_mZFLky"])
+    processed_target_neg = len(output_dataset[0].dataset["Label_mZFLky"]) - processed_target_pos
```

Comment:
Replace `"Label_mZFLky"` with a programamtic way to obtain that string. Like `model.config.output_feature[0][PROC_COLUMN]` or something like that

Reply:
Done